src/events/RetrieveBlackDuckRiskProfileOnStatus.ts

```diff
@@ -0,0 +1,74 @@
+import {
+    EventFired,
+    EventHandler,
+    failure,
+    GraphQL,
+    HandleEvent,
+    HandlerContext,
+    HandlerResult,
+    logger, MappedParameter, MappedParameters, Secret,
+    Success,
+    success,
+    Tags,
+} from "@atomist/automation-client";
+import axios from "axios";
+
+import {BlackDuckService, RiskProfile} from "../blackDuck/BlackDuckService";
+import * as graphql from "../typings/types";
+
+@EventHandler("retrieve Black Duck risk profile when there is a status",
+    GraphQL.subscriptionFromFile("../graphql/blackDuckStatus", __dirname))
+@Tags("blackDuck", "status")
+export class RetrieveBlackDuckRiskProfileOnStatus implements HandleEvent<graphql.BlackDuckStatus.Subscription> {
+
+    @Secret("blackDuckUser")
```

Comment:
These are not actual secrets. If we plan on running this in general, we'll need to discuss how to deal with team-specific secrets.